polyaxon_schemas/build.py

```diff
@@ -17,7 +17,7 @@
 def validate_image(image):
     if not image:
         raise ValidationError('Invalid docker image `{}`'.format(image))
-    tagged_image = image.split(':')
+    tagged_image = image.rsplit(':', 1)
```

Comment:
I think this is always true, so this is not the correct validation, maybe a different approach is needed?

We need to check the following use cases:
 * `len == 1` <==> no port && no tag
 * `len == 2` <==> we should assume it's true, this could be either a port or a tag.
 * `len == 3` <==> we should assume the url has a port and a tag.
 * `len > 3` should raise